README.md

```diff
@@ -1,6 +1,51 @@
 # spellcheck
 
-This repository contains a Docker image with the [R `spelling` package](https://cran.r-project.org/web/packages/spelling/index.html) installed.
-The role of this repository is to facilitate spell checking actions across `AlexsLemonade` repositories.
+This repository contains a GitHub action to run the [R `spelling` package](https://cran.r-project.org/web/packages/spelling/index.html).
+The role of this action is to facilitate spell checking actions across `AlexsLemonade` repositories.
 
-speeling
+Currently the action will only spell check text in Rmd and md files
+
+## Usage
+
+To use this action, create a `.github/workflows/spellcheck.yml` file in your repository with the following contents (modified to fit your exact needs):
+
+```yaml
+name: Check Spelling
+on:
+  pull_request:
+    branches:
+     - main
+
+jobs:
+  spell-check:
+    runs-on: ubuntu-latest
+    name: Spell check files
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Spell check action
+        uses: alexslemonade/spellcheck
```

Comment:
As is this does not work because it needs a tag, but it does work with `alexslemonade/spellcheck@main`.

Easy enough to pop in when we decide on the tag, which we can do in this PR or after.

Reply:
I made some fixes, so I think the move is going to be to merge this PR, tag a release, then update a couple things in a new PR.